src/service/FlywheelService/FlywheelController.ts

```diff
@@ -0,0 +1,65 @@
+import {APIClient, APIMapping} from '../../http';
+import {AxiosResponse} from 'axios';
+import {Flywheel} from '@flowfact/types';
```

Comment:
```suggestion
import {Flywheel} from './types';
```

Die Types sollten doch direkt beim Service liegen und nicht mehr in einem externen Repo.
Entsprechend müssen wir über eine Lösung des Exports an dieser Stelle nachdenken,